answers/src/main/java/com/coding/solutions/basic/LinkedListPalindrome.java

```diff
@@ -0,0 +1,35 @@
+package com.coding.solutions.basic;
+
+import java.util.List;
+import java.util.Stack;
+
+/**
+ * Created by vidhyaa on 10/28/15.
+ */
+public class LinkedListPalindrome {
+    public static boolean isLinkedListPalindrome(List<Character> inputList) {
+        if (inputList == null || inputList.size() == 0) {
+            return true;
+        }
+
+        int left = 0;
+        int right = inputList.size() / 2;
+        if (inputList.size() % 2 != 0) {
+            right = (inputList.size() / 2) + 1;
+        }
+
+        Stack<Character> stack = new Stack<Character>();
```

Comment:
Do this in place. Don't use stack. Also don't use Character. Make it templatized.